spec/models/outgoing_message_spec.rb

```diff
@@ -1795,6 +1795,32 @@
 
   end
 
+  describe '#prepare_message_for_resend' do
+    let(:outgoing_message) { FactoryBot.build(:initial_request) }
+    subject { outgoing_message.prepare_message_for_resend }
+
+    it 'raises an error if it receives an unexpected message_type' do
+      allow(outgoing_message).to receive(:message_type).and_return('fake')
+      expect{ subject }.to raise_error(RuntimeError)
+    end
+
+    it 'raises an error if it receives an unexpected status' do
+      outgoing_message.status = 'followup'
```

Comment:
'followup' is a message type so we think using one of [`OutgoingMessage::STATUS_TYPES`](https://github.com/mysociety/alaveteli/blob/develop/app/models/outgoing_message.rb#L33) would be a better example (the only example would be "ready" from that list)